sdk/sdkmodule/module.go

```diff
@@ -24,6 +24,8 @@ type module struct {
 	opts moduleOpts
 }
 
+var Empty = New()
```

Comment:
This is meaningless syntactic sugar that wastes human lookup time ("what is `Empty`?") instead of just using the obvious call `New()`).

On top of that, why do we even need to specify an empty module when calling `sdkintegrations.NewIntegration`?